.env.example

```diff
@@ -1,3 +1,13 @@
+# Random generation ===========================================================
+# If you change this, check the collision probability https://zelark.github.io/nano-id-cc/
+NORAY_OID_LENGTH=21 # For 10 id/hour, 15 trillion years
+# NORAY_RANDOM_ID_LENGTH=4 # For 10 id/hour, 2 days
```

Comment:
Minor: please update the comments as well
```suggestion
# NORAY_OID_LENGTH=4 # For 10 id/hour, 2 days
# NORAY_OID_LENGTH=5 # For 10 id/hour, 19 days
# NORAY_OID_LENGTH=6 # For 10 id/hour, 155 days
```

Reply:
Sorry for unresolving but the comments still say `NORAY_RANDOM_ID_LENGTH` instead of `NORAY_OID_LENGTH`